src/components/staking/LiValidator.vue

```diff
@@ -23,43 +29,62 @@
           :class="statusColor"
           class="data-table__row__info__container__status"
         />
-        <router-link
-          :to="{
-            name: 'validator',
-            params: { validator: validator.operator_address }
-          }"
-          class="data-table__row__info__container__name"
-        >
+        <span class="data-table__row__info__container__name">
           {{ validator.description.moniker }}
-        </router-link>
+        </span>
         <div class="data-table__row__info__container__description">
           <Bech32 :address="validator.operator_address" />
         </div>
       </div>
     </td>
-    <td class="li-validator__delegated-steak">
+    <td
+      class="li-validator__delegated-steak"
```

Comment:
Pretty sure these `li-validator__xyz` classes are never used and can be deleted.